libs/api/domains/user-profile/src/lib/userProfile.resolver.ts

```diff
@@ -72,4 +75,20 @@ export class UserProfileResolver {
   ): Promise<ConfirmResponse | null> {
     return await this.userUserProfileService.confirmEmail(input, user)
   }
+
+  @Mutation(() => UserDeviceToken)
+  async userProfileAddDeviceToken(
```

Comment:
`addUserProfileDeviceToken`

ref: https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#mutation-naming-conventions